backends/p4tools/modules/testgen/test/lib/p4info_api.h

```diff
@@ -1,13 +1,13 @@
 #ifndef BACKENDS_P4TOOLS_MODULES_TESTGEN_TEST_LIB_P4INFO_API_H_
 #define BACKENDS_P4TOOLS_MODULES_TESTGEN_TEST_LIB_P4INFO_API_H_
 
-#include "backends/p4tools/modules/testgen/test/gtest_utils.h"
+#include <gtest/gtest.h>
 
-namespace P4::Test {
+namespace P4::P4Tools::Test {
 
 /// Helper methods to build configurations for P4RuntimeApi Tests.
-class P4RuntimeApiTest : public P4ToolsTest {};
+class P4RuntimeApiTest : public testing::Test {};
```

Comment:
Here you are not using `::testing`. At the very least, this should be consistent. I suggest not explicitly refering to the global namespace unless there is a need to do so.